app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,4 @@
 class PostsController < ApplicationController
- 
   def new
     @post = Post.new
```

Comment:
* Kindly Create a method called `current_user` in your `ApplicationController` to make current user data available to all controllers as mentioned in the [project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md#:~:text=Create%20a%20method,the%20upcoming%20projects.)
* Also Create a Post on behalf of the `current_user `(use the method that you created in your `ApplicationController`) as mentioned in the [project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md#:~:text=Creates%20a%20Post%20on%20behalf%20of%20the%20current_user%20(use%20the%20method%20that%20you%20created%20in%20your%20ApplicationController).)